### DISCUSSION TITLE
Integración CAPTA para flujos de correspondencia

### DISCUSSION THREAD

**Discussion by NestorFGG** [SINCOSOFT-SAS/SGD#45] [General]
Se buscará una integración a través de los flujos creados por CAPTA para la gestión de correspondencia entrante o intentos de los clientes, generando flujos más flexibles y nuevas aplicaciones del mismo dentro de SINCO SGD.

**Comment by aleja550** [comment]
Nunca se realizo ni tampoco está en planeación del producto actual.
